src/lib/Map.svelte

```diff
@@ -21,17 +21,17 @@
 		DRINKS: {
 			value: 'drinks',
 			title: '飲み物',
-			icon: { id: 'icon-bottle', file: 'icon-bottle.webp' }
+			icon: { id: 'icon-bottle', file: 'icon-bottle.webp', color: 'blue' }
 		},
 		BREAD: {
 			value: 'bread',
 			title: 'パン',
-			icon: { id: 'icon-bread', file: 'icon-baguette.webp' }
+			icon: { id: 'icon-bread', file: 'icon-baguette.webp', color: 'yellow' }
```

Comment:
みにくいのでmergeする前にこれだけ変更してもらえると……！
```suggestion
			icon: { id: 'icon-bread', file: 'icon-baguette.webp', color: 'orange' }
```